src/slider.ts

```diff
@@ -68,7 +68,7 @@ export class Slider extends Component<SliderOptions> {
 
   constructor(el: HTMLElement, options: Partial<SliderOptions>) {
     super(el, options, Slider);
-    (this.el as any).M_Slider = this;
+    this.el['M_Slider'] = this;
```

Comment:
I think we might need to revert this, since it introduces backwards compatibility issues with applications using the previous object orientated approach, propose to just set it to warnings until we can declare the correct type definition